docs/content/en/docs/install/install.md

```diff
@@ -135,37 +140,6 @@ For more information,see
 * The [helm-charts](https://github.com/keptn/lifecycle-toolkit/blob/main/chart/README.md) page
```

Comment:
we should not point to the main version of the chart readme but rather to the published one here 
https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn/README.md
and the 3 subchart ones here:
https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn-lifecycle-operator/README.md
https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn-metrics-operator/README.md
https://github.com/keptn/lifecycle-toolkit-charts/blob/main/charts/keptn-cert-manager/README.md

Reply:
For now we can add this basic info and follow up with proper description of installation variation